File: app/src/main/java/com/webkul/mobikul/odoo/model/product/AddToCartResponse.java
package com.webkul.mobikul.odoo.model.product;

import android.os.Parcel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.webkul.mobikul.odoo.model.BaseResponse;

/**
 * Created by shubham.agarwal on 2/6/17.
 */

public class AddToCartResponse extends BaseResponse {

    @SuppressWarnings("unused")
    private static final String TAG = "AddToCartResponse";
    @SerializedName("productName")
    @Expose
    private String productName;


    protected AddToCartResponse(Parcel in) {
        super(in);
    }


    public String getProductName() {
        if (productName == null) {
            return "";
        }
        return productName;
    }
}

File: app/src/main/java/com/webkul/mobikul/odoo/data/response/models/AddToCartResponse.kt
package com.webkul.mobikul.odoo.data.response.models

import com.google.gson.annotations.SerializedName

data class AddToCartResponse(
    @SerializedName("product_id") val productId: Int,
    @SerializedName("line_id") val lineId: Int,
    @SerializedName("quantity") val quantity: Int,
    @SerializedName("cart_count") val newCartCount: Int,
    @SerializedName("option_ids") val optionsId: ArrayList<Int>,
    @SerializedName("warning") val warning: String
)